doc/content/examples/bibfile.bib

```diff
@@ -0,0 +1,12 @@
+@article{Abdou2021,
+doi = {10.1088/1741-4326/abbf35},
+url = {https://dx.doi.org/10.1088/1741-4326/abbf35},
+year = {2021},
+publisher = {IOP Publishing},
+volume = {61},
+number = {1},
+pages = {013001},
+author = {Mohamed Abdou and Marco Riva and Alice Ying and Christian Day and Alberto Loarte and L.R. Baylor and Paul Humrickhouse and Thomas F. Fuerst and Seungyon Cho},
```

Comment:
```suggestion
author = {Mohamed Abdou and Marco Riva and Alice Ying and Christian Day and Alberto Loarte and Larry R. Baylor and Paul Humrickhouse and Thomas F. Fuerst and Seungyon Cho},
```

Reply:
I didn't realize Larry was on this paper! 😄